emanifest-py/README.md

```diff
@@ -14,140 +14,218 @@ refactoring.
 ## Contents
 
 - [Requirements](#requirements)
-- [Dependencies](#dependencies)
 - [Installation](#installation)
-- [Usage](#usage)
-    - [Getting Started](#getting-started)
+- [Getting Started](#getting-started)
+    - [Authentication](#authentication)
     - [Methods](#methods)
-    - [Help](#help)
+    - [Regulator Usage](#regulators)
+    - [Advanced Usage](#advanced-usage-and-help)
 - [Contact](#contact)
 - [License](LICENSE.txt)
 
 ## Requirements
 
 - Python 3.7
 
-## Dependencies
-
-- requests
-- requests_toolbelt
-
 ## Installation
 
-**emanifest** can be installed directly from the Python package directory using pip:
+**emanifest** can be installed directly from the Python package Index (PyPI) using pip:
 
-```bash
-pip install emanifest
+```shell
+$ pip install emanifest
 ```
 
-## Usage
-
-### Getting Started
+## Getting Started
 
 Before using the **emanifest** package, ensure you have a RCRAInfo user account and
 the [necessary permissions](https://www.epa.gov/e-manifest/frequent-questions-about-e-manifest#user_question6) to
 generate an API ID and key.
 
-All methods to access the e-Manifest APIs are implemented by the `RcrainfoClient` class which needs to be authenticated
-with your API ID and Key. A new instance of the class can be initiated with the `new_client()` convenience function
-like so:
+All methods to access the e-Manifest APIs are implemented by the `RcrainfoClient` class which needs your API ID and Key
+to authenticate . A new instance of the class can be initiated like so:
```

Comment:
Remove extra space after "authenticate" in line 45

Reply:
complete